.circleci/config.yml

```diff
@@ -67,6 +95,64 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  e2e-simple-cni:
+    <<: *integrationDefaults
+    environment:
+            - KUBECONFIG: /go/out/minikube.conf
+            - TEST_ENV: minikube-none
+            - GOPATH: /go
+    steps:
+      - <<: *initWorkingDir
+      - checkout
+      - attach_workspace:
+          at:  /go
+      - <<: *markJobStartsOnGCS
+      - run: make sync
+      - run: make build
+      - run:
+          command: |
+            if [ ! -f /go/out/linux_amd64/release/istio-cni ]; then
+              # Should only happen when re-running a job, and the workspace is gone
+              time make build
+            fi
+            make docker.all
+      - run: docker images
+      - run:
+          name: Get Istio
+          shell: /bin/bash
+          working_directory: /go/src/istio.io/istio
+          command: |
+            git clone https://github.com/tiswanso/istio.git .
```

Comment:
We should pick a specific Istio tag here (e.g. 1.0.2) and use istio/istio not a private repo.   I see below you don't build the Istio containers but pull from the master-latest-daily.  That is risking because when the test code and associated containers aren't in lock step failures are possible and probably over time.

Reply:
Agreed... but to do initial job setup and get things working I'm using a private repo to figure out all required stuff for istio/istio.

Reply:
Since the other changes merged should be able to pull master.  Even better is if there is a tag corresponding to the master-latest-daily images we are pulling.